payroll/models/hr_payslip_line.py

```diff
@@ -61,10 +61,14 @@ def _compute_parent_line_id(self):
                     and record.slip_id == line.slip_id
                 )
                 if parent_line and len(parent_line) > 1:
-                    raise UserError(
-                        _("Recursion error. Only one line should be parent of %s")
-                        % line.parent_rule_id.name
+                    msg = (
+                        self.env._(  # pylint: disable=translation-not-lazy
+                            "Recursion error. Only one line should be parent of "
+                            "%(rule)s"
+                        )
+                        % {"rule": line.parent_rule_id.name}  # noqa: UP031  # pylint: disable=translation-not-lazy
                     )
```

Comment:
Again, I have concerns with the noqa comments, and feel they are not really required